dev/README_RELEASE_AIRFLOW.md

```diff
@@ -1041,6 +1042,16 @@ Preparing a release that is not in the latest branch:
 breeze release-prod-images --airflow-version "${VERSION}" --slim-images --skip-latest
 ```
 
+## Verify production images
+
+```shell script
+for PYTHON in 3.7 3.8 3.9 3.10
+do
+    docker pull apache/airflow:${VERSION}-python${PYTHON}
+    breeze verify-prod-image --image-name apache/airflow:${VERSION}-python${PYTHON}
+done
```

Comment:
```suggestion
done
docker pull apache/airflow:${VERSION}
breeze verify-prod-image --image-name apache/airflow:${VERSION}
```

Also worth verifying the bare version tag?

Reply:
This is just an alias. Makes no sense to run tests for it I think.

Reply:
Should at least pull then to make sure the tag was pushed?

Reply:
Why not :)